feedvalidator.py

```diff
@@ -674,6 +674,9 @@ def ParseCommandLineArguments():
   parser.set_defaults(manual_entry=True, output='validation-results.html',
                       memory_db=False, check_duplicate_trips=False,
                       limit_per_type=5, latest_version='',
+                      error_types_ignore_list='UnknownFile,UnrecognizedColumn,InvalidValue,StopTooFarFromParentStation,'
+                                              'UnusedStop,StationsTooClose,StopsTooClose,TooFastTravel,'
+                                              'TooManyConsecutiveStopTimesWithSameTime',
```

Comment:
If it's only a configuration change, could we add this to our Makefile instead?

Reply:
Yes, we can.